### PR TITLE
*: support 'admin check table' on clustered index table

### PR DESCRIPTION
**Pull request by tangenta**
### What problem does this PR solve?

Problem Summary:

support 'admin check table' on clustered index table.

### What is changed and how it works?

Proposal: [xxx](url) 

What's Changed:

How it Works:

### Check List 

Tests 

- Unit test
- Integration test

Side effects

### Release note 

- No release note

### CODE REVIEW

File: planner/core/physical_plans.go [hunk 1]
```diff
@@ -432,8 +432,9 @@ func (ts *PhysicalTableScan) Clone() (PhysicalPlan, error) {
 	}
 	clonedScan.Columns = cloneColInfos(ts.Columns)
 	clonedScan.Ranges = cloneRanges(ts.Ranges)
-	if ts.pkCol != nil {
-		clonedScan.pkCol = ts.pkCol.Clone().(*expression.Column)
+	clonedScan.PkCols = make([]*expression.Column, len(ts.PkCols))
```
Comment (coocood) [inline planner/core/physical_plans.go:435]: There is a method `cloneCols`.

**Comment by coocood** [comment]
LGTM

File: executor/executor_test.go [hunk 2]
```diff
@@ -3408,6 +3408,16 @@ func (s *testSuite) TestCheckTable(c *C) {
...
 }
 
+func (s *testSuite) TestCheckTableClusterIndex(c *C) {
+	tk := testkit.NewTestKit(c, s.store)
+
+	tk.MustExec("use test")
+	tk.MustExec("drop table if exists admin_test;")
+	tk.MustExec("create table admin_test (c1 int, c2 int, c3 int default 1, primary key (c1, c2), index (c1), unique key(c2));")
+	tk.MustExec("insert admin_test (c1, c2) values (1, 1), (2, 2), (3, 3);")
+	tk.MustExec("admin check table admin_test;")
```
Comment (lzmhhh123) [inline executor/executor_test.go:3418]: `set @@tidb_enable_clustered_index = 1` for this test?

File: planner/core/explain.go [hunk 3]
```diff
@@ -186,8 +186,17 @@ func (p *PhysicalTableScan) AccessObject() string {
...
 	buffer := bytes.NewBufferString("")
-	if p.pkCol != nil {
-		fmt.Fprintf(buffer, "pk col:%s, ", p.pkCol.ExplainInfo())
+	for i, pkCol := range p.PkCols {
+		var fmtStr string
+		switch i {
+		case 0:
+			fmtStr = "pk cols: {%s, "
+		case len(p.PkCols) - 1:
+			fmtStr = "%s}"
```
Comment (lzmhhh123) [inline planner/core/explain.go:195]:
```suggestion
			fmtStr = "pk cols: (%s, "
		case len(p.PkCols) - 1:
			fmtStr = "%s)"
```

Keep the same format for index columns in the explain info.

**Comment by tangenta** [comment]
Is GitHub broken? My changes are not synced.

**Comment by tangenta** [comment]
/run-all-tests
```
privileges_test.go:79:
     testleak.AfterTest(c)()
 /home/jenkins/agent/workspace/tidb_ghpr_unit_test/go/src/github.com/pingcap/tidb/util/testleak/leaktest.go:166:
     c.Errorf("Test %s check-count %d appears to have leaked: %v", c.TestName(), cnt, g)
 ... Error: Test  check-count 50 appears to have leaked: github.com/pingcap/tidb/domain.(*Domain).globalBindHandleWorkerLoop.func1(0xc000840a20)
```

**Comment by tangenta** [comment]
/run-sqllogic-test-2 
```
java.nio.channels.ClosedChannelException
```

**Pull request merged**